SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocolListener.h

```diff
@@ -9,7 +9,8 @@
 @protocol SDLProtocolListener <NSObject>
 
 @optional
-- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID version:(Byte)version;
+- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID version:(Byte)version __deprecated_msg("use -handleProtocolStartSessionACK:sessionID:hashID:version: instead");
+- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID hashID:(UInt32)hashID version:(Byte)version;
```

Comment:
I'm still not happy with this because [security PR](https://github.com/smartdevicelink/sdl_ios/pull/348) is creating a different method, so in the security PR I'd have to deprecate this method. It would be better to have it follow [the security framework's changes to this protocol](https://github.com/smartdevicelink/sdl_ios/blob/feature/external_security_support/SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocolListener.h) instead.

In any case, thanks for the merge nightmare :cold_sweat: